pages/studentAssessmentExam/studentAssessmentExam.sql

```diff
@@ -4,9 +4,119 @@ SELECT
 FROM
     assessment_instances AS ai,
```

Comment:
This query is incorrect. See https://github.com/PrairieLearn/PrairieLearn/pull/2919.

I suggest you avoid all comma-joins in future.

Reply:
Thanks for pointing out. We will use JOIN ON instead of comma-join in future